cypress/integration/vesselCommunications_spec.ts

```diff
@@ -0,0 +1,71 @@
+describe("As a beacon owner, I want to register my communication details so SAR can contact me in an emergency", () => {
+  const pageUrl = "/register-a-beacon/vessel-communications";
+
+  it("requires an MMSI number if the fixed VHF checkbox is selected", () => {
+    givenIAmOnTheVesselCommunicationsPage();
```

Comment:
Is it worth wrapping the common step of `givenIAmOnTheVesselCommunicationsPage` into a `beforeEach` block?

See [docs on this](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Hooks)

Reply:
Great idea :D Done